torch/nn/modules/normalization.py

```diff
@@ -69,6 +72,142 @@ def __repr__(self):
             + ', k=' + str(self.k) + ')'
 
 
+class LayerNorm(Module):
+    r"""Applies Layer Normalization over a mini-batch of inputs as described in
+    the paper `Layer Normalization`_ .
+
+    .. math::
+        y = \frac{x - mean[x]}{ \sqrt{Var[x]} + \epsilon} * gamma + beta
+
+    The mean and standard-deviation are calculated separately over the last
+    certain number dimensions with shape specified by :attr:`normalized_shape`.
+    Gamma and beta are learnable parameters of :attr:`normalized_shape` if
+    :attr:`elementwise_affine` is ``True``.
+
+    .. note::
+        Unlike Batch Normalization and Instance Normalization, which applies
+        scalar scale and bias for each entire channel/plane with the
+        :attr:`affine` option, Layer Normalization applies per-element scale and
+        bias with :attr:`elementwise_affine`.
+
+    By default, this layer uses statistics computed from input data in both
+    training and evaluation modes.
+
+    If :attr:`track_running_stats` is set to ``True``, during training this
+    layer keeps running estimates of its computed mean and variance, which are
+    then used for normalization during evaluation. The running estimates are
+    kept with a default :attr:`momentum` of 0.1.
+
+    .. note::
+        This :attr:`momentum` argument is different from one used in optimizer
+        classes and the conventional notion of momentum. Mathematically, the
+        update rule for running statistics here is
+        :math:`\hat{x}_\text{new} = (1 - \text{momentum}) \times \hat{x}_\text{new} + \text{momemtum} \times x_t`,
+        where :math:`\hat{x}` is the estimated statistic and :math:`x_t` is the
+        new observed value.
+
+    Args:
+        normalized_shape (list or torch.Size): input shape from an expected input of size
+            `[* x normalized_shape[0] x normalized_shape[1] x ... x normalized_shape[-1]]`
```

Comment:
Nit: you can add a comment saying that * represents any number of dimensions (though we do have it elsewhere in our docs)